qir/qsharp-target-tests/TargetTestScripts/Intrinsic/ApplyControlledX.qs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace QATSample {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Measurement;
+
+    @EntryPoint()
+    operation Main(): Result
+    {
+      use control = Qubit();
+      use target = Qubit();
+
+      ApplyControlledX(control, target);
```

Comment:
This is not a public callable, and is kept in the target package as an internal implementation detail. It is not expected to be usable directly by a developer.

Reply:
Yes, the folder `Intrinsic` needs to go - that was added by mistake.